src/healIndex.jsx

```diff
@@ -0,0 +1,450 @@
+  ReactGA.initialize(gaTracking);
+  ReactGA.pageview(window.location.pathname + window.location.search);
+  await Promise.all(
+    [
+      store.dispatch(fetchSchema),
+      store.dispatch(fetchDictionary),
+      store.dispatch(fetchVersionInfo),
+      // resources can be open to anonymous users, so fetch access:
+      store.dispatch(fetchUserAccess),
+      store.dispatch(fetchUserAuthMapping),
+      // eslint-disable-next-line no-console
+      fetchAndSetCsrfToken().catch((err) => { console.log('error on csrf load - should still be ok', err); }),
+    ],
+  );
+  // FontAwesome icons
+  library.add(faAngleUp, faAngleDown);
+
+  render(
+    <div>
+      <Provider store={store}>
+        <ThemeProvider theme={theme}>
+          <BrowserRouter basename={basename}>
+            <div>
+              {GA.init(gaTracking, dev, gaDebug) && <RouteTracker />}
+              {enableDAPTracker && <DAPRouteTracker />}
+              {isEnabled('noIndex') ?
+                <Helmet>
+                  <meta name='robots' content='noindex,nofollow' />
+                </Helmet>
+                : null
+              }
+              <ReduxTopBar />
+              <ReduxNavBar />
+              <div className='main-content'>
+                <Switch>
+                  {/* process with trailing and duplicate slashes first */}
+                  {/* see https://github.com/ReactTraining/react-router/issues/4841#issuecomment-523625186 */}
+                  {/* Removes trailing slashes */}
+                  <Route
+                    path='/:url*(/+)'
+                    exact
+                    strict
+                    render={({ location }) => (
+                      <Redirect to={location.pathname.replace(/\/+$/, '')} />
+                    )}
+                  />
+                  {/* Removes duplicate slashes in the middle of the URL */}
+                  <Route
+                    path='/:url(.*//+.*)'
+                    exact
+                    strict
+                    render={({ match }) => (
+                      <Redirect to={`/${match.params.url.replace(/\/\/+/, '/')}`} />
+                    )}
+                  />
+                  <Route
+                    exact
+                    path='/login'
+                    component={
+                      (
+                        props => (
+                          <ProtectedContent
+                            public
+                            filter={() => store.dispatch(fetchLogin())}
+                            component={ReduxLogin}
+                            {...props}
+                          />
+                        )
+                      )
+                    }
+                  />
+                  <Route
+                    exact
+                    path='/'
+                    component={
+                      props => (
+                        <ProtectedContent
+                          public={indexPublic}
```

Comment:
maybe you will want to modify the `public` policy so it is the same with discovery page

Reply:
can I just remove this - `public={indexPublic}`?